Docs: add SEO meta descriptions to each page (#1699)
VictoriaMetrics/VictoriaLogs: docs/victorialogs/data-ingestion/opentelemetry.md

---
weight: 4
title: OpenTelemetry Setup

> description: "Configure OTel SDK or Collector to send logs to VictoriaLogs"

Reviewer: ```suggestion
description: "Configure OpenTelemetry SDKs and Collectors to send logs to VictoriaLogs."
```
